.github/workflows/build.yml

```diff
@@ -4,9 +4,6 @@ on: [push, pull_request]
 jobs:
   build:
     uses: charmbracelet/meta/.github/workflows/build.yml@main
-
-  build-go-mod:
```

Comment:
This uses the go.mod version to test the package while the `build` job uses the latest stable go version

Reply:
ah, right. will undo